### ISSUE TITLE
FS22 Issues

### ISSUE THREAD

**Issue by Jayshua83** [yumi-modding/FS19_RefreshContracts#2]
This appears to work in FS22 Single Player but NOT on a Dedicated Server. Why would that be? Is there an easy fix I can make? Did it ever work on a Dedicated Server?

Thanks!

**Comment by yumi-modding** [comment]
Hello,

I think i tested FS19 version on dedicated server but to be confirmed.
I will port the mod to FS22 soon, but i cannot publish on modhub yet as requested by Giants.

Yumi

**Comment by pvm** [comment]
Yeah, same here, still not working. You press X and nothing happens

**Comment by pvm** [comment]
Also, there was some finished contracts from another farms showing to me as well. (we were without contractor active)

**Comment by yumi-modding** [comment]
Hello @pvm, @Jayshua83,

Could you please send me information on how I can join a MP game with one of you and try to understand what the problem is. It works for me if I start a MP game, I cannot test on Dedi server, I only have 1 game license.
I might have to share with you a new zip for the mod adding traces so I can see where is the issue.
You can contact me on Discord or by email.

Yumi

**Comment by Axelboss2010** [comment]
Hello Yumi,
do you still need access to a Dedicatet server?
I would like to make mine available to you.

Best regards

Axelboss2010

**Comment by yumi-modding** [comment]
> Hello Yumi, do you still need access to a Dedicatet server? I would like to make mine available to you.
> 
> Best regards
> 
> Axelboss2010

Hello

Yes you can send me an email so we can see how to proceed testing on your server.
Thanks

Yumi

**Comment by Axelboss2010** [comment]
No Problem.
Can you tell me your Email ?

**Comment by yumi-modding** [comment]
You can find it in the git logs but yes it's yumi [at] yumi-modding.fr

**Comment by Bandesss** [comment]
Wir bieten uns auch an, ohne Mod ist LS kaputt

**Comment by yumi-modding** [comment]
Hello,

I'm working with @Axelboss2010 to fix the issue with his server. I hope to have a fix soon

**Comment by Bandesss** [comment]
Ah super, komisch im Stream welche sagten bei ihnen geht's auf dem Server 
Danke für die info
